requirements.txt

```
@@ -19,6 +19,6 @@
 # IN THE SOFTWARE.
 
 # Direct dependencies
-mkdocs>=1
+mkdocs>=1 <1.2
```

```suggestion
mkdocs>=1, <1.2
```
I believe a "," is needed here: https://www.python.org/dev/peps/pep-0508/#examples